### PR TITLE
Feature/linkerd tutorial

### PR DESCRIPTION
**Pull request by ryadav88**

### CODE REVIEW

File: src/blog/2016-10-10_buoyant_service_discovery.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+The linkerd package in the DC/OS Universe repository installs a linkerd instance on each node in the cluster and configures it to act as an HTTP proxy. This means that most HTTP applications can use linkerd simply by setting the `http_proxy` environment variable to `localhost:<linkerd port>`, without code changes. (For non-HTTP applications, or situations where setting this environment variable is not viable, linkerd can still be used with [a little more configuration work](https://linkerd.io/config/0.8.1/linkerd/index.html).)
+
+Let’s walk through a quick demonstration of installing linkerd and using it for service discovery. After this step, we’ll also show you how, once it’s installed, you can also easily use linkerd to capture and display top-line service metrics like success rates and request latencies.
+
+## Installing linkerd
+
+### STEP 0: PREREQUISITES
```
Comment (wmorgan) [inline src/blog/2016-10-10_buoyant_service_discovery.md]: I think the all-caps here is a holdover from a previous formatting of the doc. You're welcome to keep them, but it's not necessary.

File: src/blog/2016-10-10_buoyant_service_discovery.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+You will need:
+- A running DC/OS cluster.
+- The [DC/OS CLI installed](https://dcos.io/docs/1.8/usage/cli/install/).
+
+### STEP 1: DEPLOY A SAMPLE APPLICATION
+First, we’ll deploy a simple example application. Use the `webapp.json` example application (borrowed from this [Marathon guide](https://mesosphere.github.io/marathon/docs/native-docker.html)) from the DC/OS CLI as follows:
```
Comment (siggy) [inline src/blog/2016-10-10_buoyant_service_discovery.md]:
```
[webapp.json](https://raw.githubusercontent.com/BuoyantIO/linkerd-examples/master/dcos/webapp.json)
```

File: src/blog/2016-10-10_buoyant_service_discovery.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+```bash
+dcos marathon app add webapp.json
```
Comment (siggy) [inline src/blog/2016-10-10_buoyant_service_discovery.md]:
```bash
dcos marathon app add https://raw.githubusercontent.com/BuoyantIO/linkerd-examples/master/dcos/webapp.json
```

File: src/blog/2016-10-10_buoyant_service_discovery.md (hunk 1 above)
Comment (ryadav88) [inline src/blog/2016-10-10_buoyant_service_discovery.md]: removing all caps

File: src/blog/2016-10-10_buoyant_service_discovery.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+title: Service Discovery and Visibility with Ease on DC/OS
+date: 2016-10-10
+author: Andrew Seigner (Buoyant), Ravi Yadav (Mesosphere)
+category: services
+layout: article.jade
+collection: posts
+lunr: true
+---
+
+Interested in seeing a live demo? Join Buoyant CEO William Morgan on Tuesday, October 11th for a live webinar and demo of everything below. [Register here to attend](http://bit.ly/2dLhnkg).
```
Comment (ryadav88) [inline src/blog/2016-10-10_buoyant_service_discovery.md]: @wmorgan lets go with what you prefer :) keeping it

**Pull request merged**